pop-api/integration-tests/src/lib.rs

```diff
@@ -46,13 +47,12 @@ fn new_test_ext() -> sp_io::TestExternalities {
 	ext
 }
 
-fn load_wasm_module<T>(path: &str) -> std::io::Result<(Vec<u8>, <T::Hashing as Hash>::Output)>
+fn load_wasm_module<T>(path: &str) -> std::io::Result<Vec<u8>>
 where
 	T: frame_system::Config,
 {
 	let wasm_binary = std::fs::read(path)?;
-	let code_hash = T::Hashing::hash(&wasm_binary);
-	Ok((wasm_binary, code_hash))
+	Ok(wasm_binary)
```

Comment:
Fine as is, but if you do another pass this could be simplified to just be 'std::fs::read(path)' as the only statement in the function. Not sure the T generic type parameter is required either.